app/assets/javascripts/student_profile_v2/attendance_details.js

```diff
@@ -60,12 +85,14 @@
 
     renderIncidentHistory: function() {
       return dom.div({}, this.props.disciplineIncidents.map(function(incident) {
-        return dom.div({ key: incident.occurred_at },
-          dom.div({}, 'Date: ' + moment.utc(incident.occurred_at).format('MMM D, YYYY')),
-          dom.div({}, 'Code:' + incident.incident_code),
-          dom.div({}, 'Location: ' + incident.incident_location),
-          dom.div({}, 'Description: ' + incident.incident_description)
-        );
+        return dom.div({ style: styles.box, key: incident.occurred_at },
+          dom.div({ sytle: styles.header },
+            dom.div({ style: styles.item }, dom.span({ style: styles.itemHead }, 'Date: '), dom.span({}, moment.utc(incident.occurred_at).format('MMM D, YYYY'))),
+            dom.div({ style: styles.item }, dom.span({ style: styles.itemHead }, 'Code: '), dom.span({}, incident.incident_code)),
+            dom.div({ style: styles.item }, dom.span({ style: styles.itemHead }, 'Location: '), dom.span({}, incident.incident_location))
+          ),
```

Comment:
If it's helpful, you can also factor out the code for making an item into method, something like:

```
renderItem: function(label, content) {
  return dom.div({ style: styles.item }, dom.span({ style: styles.itemHead }, label), dom.span({}, content));
}
```

This isn't super necessary, just pointing out how you can refactor React UI code like plain JS code. :)